src/CsharpClient/QuixStreams.Streaming/QuixApi/Portal/Workspace.cs

```diff
@@ -176,4 +181,194 @@ internal enum WorkspaceStatus
         /// </summary>
         Disabled
     }
+    
+    internal enum WorkspaceBrokerType
```

Comment:
Need "Unknown" as first, or something similar, so when can't find a matching enum, it doesn't give the wrong enum.

Also need to test it